blaze-client/src/test/scala/org/http4s/client/blaze/ClientTimeoutSpec.scala

```diff
@@ -42,44 +41,44 @@ class ClientTimeoutSpec extends Http4sSpec {
 
   "Http1ClientStage responses" should {
     "Timeout immediately with an idle timeout of 0 seconds" in {
-      val c = mkClient(new SlowTestHead(List(mkBuffer(resp)), 0.seconds), 
+      val c = mkClient(new SlowTestHead(List(mkBuffer(resp)), 0.seconds),
                        mkConnection())(0.milli, Duration.Inf)
 
-      c.fetchAs[String](FooRequest).run must throwA[TimeoutException]
+      c.fetchAs[String](FooRequest).unsafeRun() must throwA[TimeoutException]
     }
 
     "Timeout immediately with a request timeout of 0 seconds" in {
       val tail = mkConnection()
       val h = new SlowTestHead(List(mkBuffer(resp)), 0.seconds)
       val c = mkClient(h, tail)(Duration.Inf, 0.milli)
 
-      c.fetchAs[String](FooRequest).run must throwA[TimeoutException]
+      c.fetchAs[String](FooRequest).unsafeRun() must throwA[TimeoutException]
     }
 
     "Idle timeout on slow response" in {
       val tail = mkConnection()
       val h = new SlowTestHead(List(mkBuffer(resp)), 10.seconds)
       val c = mkClient(h, tail)(1.second, Duration.Inf)
 
-      c.fetchAs[String](FooRequest).run must throwA[TimeoutException]
+      c.fetchAs[String](FooRequest).unsafeRun() must throwA[TimeoutException]
     }
 
     "Request timeout on slow response" in {
       val tail = mkConnection()
       val h = new SlowTestHead(List(mkBuffer(resp)), 10.seconds)
       val c = mkClient(h, tail)(Duration.Inf, 1.second)
 
-      c.fetchAs[String](FooRequest).run must throwA[TimeoutException]
+      c.fetchAs[String](FooRequest).unsafeRun() must throwA[TimeoutException]
     }
 
     "Request timeout on slow POST body" in {
 
       def dataStream(n: Int): EntityBody = {
-        implicit def defaultSecheduler = DefaultTimeoutScheduler
+        implicit val defaultScheduler = Scheduler.fromFixedDaemonPool(2)
```

Comment:
I think this is going to leak thread pools in our tests.  Where is this being shut down?

There's also an `Http4s.TestPool` defined.